va/gsb.go

```diff
@@ -24,28 +26,34 @@ type SafeBrowsing interface {
 // third-party safe browsing API. It's meant be called by the RA before pending
 // authorization creation. If no third-party client was provided, it fails open
 // and increments a Skips metric.
-func (va *ValidationAuthorityImpl) IsSafeDomain(ctx context.Context, req *core.IsSafeDomainRequest) (*core.IsSafeDomainResponse, error) {
+func (va *ValidationAuthorityImpl) IsSafeDomain(ctx context.Context, req *vaPB.IsSafeDomainRequest) (*vaPB.IsDomainSafe, error) {
+	if req == nil || req.Domain == nil {
+		return nil, bgrpc.ErrMissingParameters
+	}
+	var status bool
```

Comment:
Why declare this at the top scope? Better to declare it in each scope that it's needed, so it's clear that there's not state being passed along through the execution of the function.